hack/quickstart/init-master.sh

```diff
@@ -7,6 +7,7 @@ CLUSTER_DIR=${CLUSTER_DIR:-cluster}
 IDENT=${IDENT:-${HOME}/.ssh/id_rsa}
 SSH_OPTS=${SSH_OPTS:-}
 SELF_HOST_ETCD=${SELF_HOST_ETCD:-false}
+CLOUD_PROVIDER=${CLOUD_PROVIDER:-}
```

Comment:
Ok, I'm fine with disabling it explicitly (as it is now) when not provided. Auto-detection doesn't feel right to me.